### PR TITLE
Feature/react16

### PR DESCRIPTION
**Pull request by McDoyen**
* Add react 16 support
* Add travis CI support

### CODE REVIEW

File: src/components/ProgressCircleContainer.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class ProgressCircleContainer extends Component<ContainerProps, C
 
     render() {
         if (this.state.showAlert) {
-            return createElement(Alert, { message: this.state.alertMessage });
+            return createElement(Alert, { bootstrapStyle: "danger", message: this.state.alertMessage });
```
Comment (edwinmp) [inline src/components/ProgressCircleContainer.ts]: Add a widget specific class name `widget-progress-circle-alert`

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,31 @@
+0 info it worked if it ends with ok
```
Comment (edwinmp) [inline npm-debug.log]: Remove this file please. Add it to your global .gitignore

File: README.md [hunk 3]
```diff
@@ -35,7 +40,7 @@ To set up the development environment, run:
 Create a folder named `dist` in the project root.
 
 
-Create a Mendix test project in the dist folder and rename its root folder to `dist/MxTestProject`. Or get the test project from https://github.com/mendixlabs/progress-circle/releases/download/v1.2.2/TestProgressCircle.mpk Changes to the widget code shall be automatically pushed to this test project.
+Create a Mendix test project in the dist folder and rename its root folder to `dist/MxTestProject`. Or get the test project from https://github.com/mendixlabs/progress-circle/releases/latest Changes to the widget code shall be automatically pushed to this test project.
```
Comment (edwinmp) [inline README.md]: Make a link `[https://github.com/mendixlabs/progress-circle/releases/latest](https://github.com/mendixlabs/progress-circle/releases/latest)`

**Review by edwinmp — CHANGES_REQUESTED** [review]
Small stuff.

File: src/components/ProgressCircleContainer.ts (hunk 1 above)
Comment (akileng56) [reply src/components/ProgressCircleContainer.ts]: Done

File: npm-debug.log (hunk 2 above)
Comment (akileng56) [reply npm-debug.log]: Removed

File: README.md (hunk 3 above)
Comment (akileng56) [reply README.md]: Done

**Pull request merged**